.github/workflows/compliance.yml

```diff
@@ -13,3 +13,14 @@ permissions:
 jobs:
   compliance:
     uses: open-sauced/hot/.github/workflows/compliance.yml@main
+name: reviewdog
+on: [pull_request]
+
+jobs:
+  vale:
+    name: runner / vale
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: errata-ai/vale-action@reviewdog
+         files: "* .md"
```

Comment:
This should be:

```yaml
- uses: errata-ai/vale-action@reviewdog
  with:
    files: "* .md"
```

And it looks like the value for files should be a path: https://github.com/errata-ai/vale-action#files-default-all